src/brokers/broker-pods/PodsList.container.tsx

```diff
@@ -79,7 +64,33 @@ const PodsContainer: FC = () => {
             {t('broker')} {name}
           </Title>
         </div>
-        {!loading && brokerPods.length > 0 && (
+        {loading && (
+          <EmptyState>
+            <EmptyStateIcon variant="container" component={Spinner} />
+            <Title size="lg" headingLevel="h4">
+              Loading
+            </Title>
+          </EmptyState>
+        )}
+        {brokerPods.length === 0 && (
+          <EmptyState>
+            <EmptyStateIcon icon={SearchIcon} />
+            <Title size="lg" headingLevel="h4">
+              No results found. Check the status of the deployment.
+            </Title>
+            <EmptyStateBody>No results match.</EmptyStateBody>
+          </EmptyState>
+        )}
+        {loadError && (
```

Comment:
Hello @lavocatt , I noticed that when there's a load error, both the loading and error states are being displayed simultaneously. It would be more user-friendly to display only the load error state directly right. Could you please update the conditions to ensure only the error state is shown in such cases.

![Screenshot from 2024-06-21 12-39-17](https://github.com/artemiscloud/activemq-artemis-self-provisioning-plugin/assets/99248895/8303bfee-6bad-468c-b644-f5d93ccd2cd6)

Reply:
Good catch thanks! I'll fix this

Reply:
Should be fixed now :)